config/routes.rb

```diff
@@ -20,6 +20,7 @@
     resources :regional_strategies, only: %i[show]
     resources :reviews, only: :show
     resources :thematic_reviews, only: %i[index show]
+    resources :uk_strategies, only: :show
```

Comment:
Curiosity .. thematic reviews, regional strategies and now uk strategies they all have underscores. @margOnline / @your Is that what the business asked? I didn't read anything mentioned on the card, so I am assuming there is no definition of what the url should be?

Reply:
@tomas-stefano @margOnline told me that this "issue" with the urls (e.g. the slug repeating word present in the resource path name) is known and for now we should carry on with what we have, maybe we'll revisit it once the business takes a clear decision on it (I think, correct me if I am wrong).